wof/examples/flask/odm2/timeseries/runserver_odm2_timeseries.py

```diff
@@ -2,9 +2,6 @@
 
 import wof.flask
 
-curr_folder = os.path.abspath(os.path.dirname(__file__))
-sys.path.insert(0, '%s/../../../' % curr_folder)
```

Comment:
I'm not sure. @lsetiawan, can you review this please? Though I realize you may not have time today (Don usually works on a different project on Thursdays). Please take a quick look to see if this is something you can decide on today, or if it'll have to wait till tomorrow. And let us know -- @ocefpaf and I are actively working through WOFpy PR's right now / today!